services/QuillLMS/lib/tasks/temporary/translate.rake

```diff
@@ -7,106 +7,318 @@ namespace :translate do
     count = hints.count
     hints.each_with_index do |hint, index|
       puts "translating #{index + 1}/#{count}..."
-      res = Evidence::OpenAI::Translate.run(english_text: hint.description, prompt: hint.prompt)
-      hint.create_translation_mappings
-      hint.english_texts.first.openai_translated_texts.create(
-        translation: res,
-        locale: Translatable::DEFAULT_LOCALE)
+      hint.translate!
     end
 
   end
 
-  task activities: :environment do
-    activities = Activity.where("id in (?)", activity_ids).limit(50)
-    CSV.open("open_ai_spanish_activities.csv", "wb") do |csv|
-      csv << ["english", "spanish"]
-      activities.each_with_index do |activity, index|
-        puts "translating #{index + 1}/50..."
-        html = activity.data["landingPageHtml"]
-        next unless html.present?
+  desc 'translate activities and their questions'
+  task :activities, [:limit] => :environment do |t, args|
```

Comment:
Not worth dealing with in this PR, but it may be worth considering writing some of our future tasks in `thor` instead of `rake`.  We've been using more and more `thor` recently as it generally has much cleaner syntax for taking command line args.  That said, this is totally reasonable, and no reason to change approaches in the middle of a project here.